exercises/resistor-color/example.go

```diff
@@ -0,0 +1,70 @@
+package resistorcolor
+
+import (
+	"regexp"
+	"strings"
+)
+
+type Code struct {
+	name string
+	code int
+}
+
+var colors = Code{
```

Comment:
I think this is supposed to be:
```suggestion
var colors = []Code{
```